clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -195,6 +260,35 @@ void zoo() {
     a[i] = 0;
 }
 
+// Test for Intel FPGA loop attributes compatibility
+void woo() {
```

Comment:
Maybe it would be nice to get the more understandable name to this function. I.e. intelfpga_loop_attrs_compatibility()